util/internal/integrationtest/mapper/orm/src/main/resources/hibernate.properties

```diff
@@ -25,3 +25,10 @@ hibernate.cache.region.factory_class = org.hibernate.testing.cache.CachingRegion
 
 hibernate.implicit_naming_strategy = component-path
 
+# Temporary workaround for:
+# - https://hibernate.atlassian.net/browse/HHH-15054
+#   (affects org/hibernate/search/test/proxy/ProxyTest)
+# - https://hibernate.atlassian.net/browse/HHH-15058
+#   (affects tests in org/hibernate/search/integrationtest/mapper/orm/automaticindexing/association/bytype/manytomany,
+#   in particular)
+hibernate.mapping.default_list_semantics=bag
```

Comment:
I think this is no longer necessary since this is the default since 6.0.0.CR2: https://github.com/hibernate/hibernate-orm/commit/7aaeebe3af8344d9f49718047b0624b6fa0b8ef9